Dockerfile

```diff
@@ -68,6 +75,8 @@ COPY cli/src/semgrep/semgrep_interfaces cli/src/semgrep/semgrep_interfaces
 #    tools like gcc, make, which are not provided by default on Alpine.
 #
 # An alternative to ocaml-layer would be to use https://depot.dev/
+# update: we actually started to use depot.dev to speedup multi-arch (arm)
+# docker image, so maybe we could use it to get rid of ocaml-layer
```

Comment:
we definitely could but would need to think a little more about opam lockfiles (or something equivalent) so that we're not over- or under-caching